resources/overwrites/cime/src/components/data_comps_mct/datm/cime_config/namelist_definition_datm.xml

```diff
@@ -446,13 +446,12 @@
       <value stream="CLM1PT.1x1_VIKE">$DIN_LOC_ROOT_CLMFORC/VIKE</value>
       <value stream="CLM1PT.1x1_JOAS">$DIN_LOC_ROOT_CLMFORC/JOAS</value>
       <value stream="CLM1PT.1x1_LIAH">$DIN_LOC_ROOT_CLMFORC/LIAH</value>
-      <value stream="CLM1PT.1x1_FINN">$DIN_LOC_ROOT_CLMFORC/FINN</value>
+      <value stream="CLM1PT.1x1_ISK">$DIN_LOC_ROOT_CLMFORC/ISK</value>
```

Comment:
It seems like we have already added the ISK site in this file on line 452, and many other relevant files.

Reply:
Good catch! I'll remove the duplicates.